src/com/khadri/hibernate/client/ClientMain.java

```diff
@@ -0,0 +1,36 @@
+package com.khadri.hibernate.client;
+
+
+import org.hibernate.Session;
+import org.hibernate.SessionFactory;
+import org.hibernate.Transaction;
+import org.hibernate.cfg.Configuration;
+
+import com.khadri.hibernate.entity.Employee;
+
+
+public class ClientMain {
+
+	public static void main(String[] args) {
+		Configuration cfg = new Configuration();
```

Comment:
Separate code changes of session creation to the file BasicOperationSession.java
give package name as com.khadri.hibernate.session

Reply:
completed the changes